src/Uno.UI.Runtime.Skia.Gtk/GtkHost.cs

```diff
@@ -10,11 +10,10 @@
 using GtkApplication = Gtk.Application;
 using WinUIWindow = Windows.UI.Xaml.Window;
 using Uno.UI.Runtime.Skia.Gtk.Extensions;
-using Uno.UI.Runtime.Skia.UI.Xaml.Controls;
 using Uno.UI.Runtime.Skia.Gtk.UI.Controls;
 using Uno.UI.Runtime.Skia.Gtk.UI;
 
-namespace Uno.UI.Runtime.Skia
+namespace Uno.UI.Runtime.Skia.Gtk
 {
 	public partial class GtkHost : ISkiaApplicationHost
```

Comment:
@jeromelaban Is this breaking change okay? If so, I'll doc it as well. Otherwise, I'll revert.

It may require template changes.

Reply:
I think it is worth doing now before 5.0. My only concern is potential clash against GTK's `Gtk` namespace, but it should not be a huge issue